src/__tests__/nonstandard.js

```diff
@@ -4,14 +4,32 @@ test('non-standard selector', '.icon.is-$(network)', (t, tree) => {
     let class1 = tree.nodes[0].nodes[0];
     t.deepEqual(class1.value, 'icon');
     t.deepEqual(class1.type, 'class');
+    t.deepEqual(class1.source.start.column, 1);
+    t.deepEqual(class1.source.end.column, 5);
+    t.deepEqual(class1.sourceIndex, 0);
+
     let class2 = tree.nodes[0].nodes[1];
     t.deepEqual(class2.value, 'is-$(network)');
     t.deepEqual(class2.type, 'class');
+    t.deepEqual(class2.source.start.column, 6);
+    // t.deepEqual(class2.source.end.column, 19); // Fail - 10
```

Comment:
This still fails because the `splitWords` method processes this selector as `[".icon", ".is-$"]` and there is another place in the code that adds the rest of the selector in (I believe it's the pseudoselector code). I'm not sure how to fix this.